fusequery/query/src/datasources/system/crashme_table.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
+use std::any::Any;
+use std::sync::Arc;
+
+use common_datavalues::DataField;
+use common_datavalues::DataSchemaRef;
+use common_datavalues::DataSchemaRefExt;
+use common_datavalues::DataType;
+use common_exception::Result;
+use common_planners::Part;
+use common_planners::ReadDataSourcePlan;
+use common_planners::ScanPlan;
+use common_planners::Statistics;
+use common_streams::SendableDataBlockStream;
+
+use crate::datasources::Table;
+use crate::sessions::FuseQueryContextRef;
+
+pub struct CrashMeTable {
+    schema: DataSchemaRef,
+}
+
+impl CrashMeTable {
+    pub fn create() -> Self {
+        CrashMeTable {
+            schema: DataSchemaRefExt::create(vec![DataField::new("crash", DataType::Int64, false)]),
+        }
+    }
+}
+
+#[async_trait::async_trait]
+impl Table for CrashMeTable {
```

Comment:
Better name to CrashMyself ?

Reply:
Have changed the `system.crashme` table to  `crashme` function.
Hmm, `crashme` short than `crashmyself`, this function is for testing only